optaplanner-docs/src/main/asciidoc/DesignPatterns/DesignPatterns-chapter.adoc

```diff
@@ -11,93 +11,73 @@
 [[designPatternsIntroduction]]
 == Design Patterns Introduction
 
-These design patterns list and solve common design challenges.
-
+OptaPlanner design patterns are generic reusable solutions to common challenges in the model or architecture of projects that perform constraint solving. The design patterns in this section list and solve common design challenges.
 
 [[domainModelingGuide]]
-== Domain Modeling Guide
-
-A good model can greatly improve the success of your planning implementation.
-These guidelines help to design a good model:
-
-. *Draw a class diagram* of your domain model. _Normalize_ it get rid of duplicate data.
-** Write down some sample instances for each class.
-*** For example in employee rostering, the samples for the `Employee` class are _Ann_, _Bert_ and _Carl_.
-
-. *Determine which relationships (or fields) change during planning*. Color them orange.
-Often there is only one such relationship (or field).
-One side of these relationships will become a planning variable later on.
-** For example in employee rostering, the `Shift` to `Employee` relationship changes during planning,
-so it is orange.
-Other relationships, such as from `Employee` to `Skill`, are immutable during planning
-because Planner can't decide to give an employee an extra skill.
-
-. If there are multiple relationships (or fields): *check for <<shadowVariable,shadow variables>>*.
-A shadow variable does change during planning,
-but its value can be calculated based on one or more genuine planning variables, without dispute.
-Color those shadow relationships (or fields) purple.
-** Only one side of a bi-directional relationship can be a genuine planning variable,
-the other side will become an <<bidirectionalVariable,inverse relation shadow variable>> later on.
-Keep those relationships in orange.
+== Domain Modeling Guidelines
+
+Follow the guidelines listed in this section to create a well thought-out model that can contribute significantly to the success of your planning.
+
+
+. *Draw a class diagram of your domain model.*
+.. Make sure there are no duplications in your data model and that relationships between objects are clearly defined.
+
+.. Create sample instances for each class. For example, in the Employee Rostering starter application, the samples for the `Employee` class are `Ann`, `Bert`, and `Carl`.
```

Comment:
"Employee Rostering starter application" is ambiguous: it can be the nurse rostering swing application, or optaweb-employee-rostering.
Neither has datasets with the names Ann, Beth and Carl. But those names make more sense that the names in those apps (they are nurse 1, nurse 2, nurse 3, etc)

Reply:
I do this often in the rest of the docs: explain a concept and than write "for example ..."
I avoid the word samples - what does the ibm style guideline book recommend?

Reply:
Geoffrey, here is the original text, that I edited, so you can see I just reused what was there:

Draw a class diagram of your domain model. Normalize it get rid of duplicate data.
Write down some sample instances for each class.
For example in employee rostering, the samples for the Employee class are Ann, Bert and Carl.

The IBM SG doesn't comment directly about 'sample' or 'example' but does refer to both and thereby differentiating the two. I would say that a sample is a concrete 'thing' where an example is template that you might use to create a a thing, possibly even a sample.

I suggest we rewrite as follows:
.. Create instances for each class. For example, in employee rostering  `Employee` class, create `Ann`, `Bert`, and `Carl`.

Unless of course I've changed the meaning. Please feel free to suggest the wording you prefer.

Reply:
Good idea. My only issue was with the suffice "starter application". Without that, it doesn't imply that the example to illustrate the concepts actually exists as code too our distribution zip.